aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -185,6 +189,23 @@ module aptos_framework::code {
             request_publish(addr, module_names, code, policy.policy)
     }
 
+    public fun is_package_upgradeable(metadata_serialized: vector<u8>): bool {
+        let metadata = util::from_bytes<PackageMetadata>(metadata_serialized);
+        metadata.upgrade_policy.policy < upgrade_policy_immutable().policy
```

Comment:
This feels a little brittle for the future, could we just check explicitly whether it matches one of the appropriate variants?

Reply:
Removing as discussed above.